_migrations/migration-console/accessing-the-migration-console.md

```diff
@@ -32,10 +29,6 @@ export SERVICE_NAME=migration-console
 export TASK_ARN=$(aws ecs list-tasks --cluster migration-${STAGE}-ecs-cluster --family "migration-${STAGE}-${SERVICE_NAME}" | jq --raw-output '.taskArns[0]')
 aws ecs execute-command --cluster "migration-${STAGE}-ecs-cluster" --task "${TASK_ARN}" --container "${SERVICE_NAME}" --interactive --command "/bin/bash"
 ```
-</details>
-
-## Troubleshooting
 
-### Deployment Stage
 
-Typically, `STAGE` is `dev`, but this may vary based on what the user specified during deployment.
\ No newline at end of file
+Typically, `STAGE` is  equivalent to a standard `dev` environment, but this may vary based on what the user specified during deployment.
```

Comment:
```suggestion
Typically, `STAGE` is equivalent to a standard `dev` environment, but this may vary based on what the user specified during deployment.
```